client/lcd/swaggerui/swagger.json

```diff
@@ -15,7 +15,6 @@
       "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
     }
   },
-  "host": "localhost:1317",
   "basePath": "/",
```

Comment:
Remove this field in `swagger.json`, then the swagger-ui will automatically use the correct host IP and port to send request. I have verified this.